package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/index.js",
   "dependencies": {
     "@aws-amplify/api": "^1.3.3",
+    "@rjsf/core": "^2.2.1",
```

Comment:
`@rjsf/core` is the new version of `react-jsonschema-form`. I don't think we can use the new version because we are using `react-jsonschema-form-extras`, which depends on the older package `react-jsonschema-form`.

Reply:
Reverted dependencies for  `react-jsonschema-form`.